examples/cerebro/list.go

```diff
@@ -76,6 +85,165 @@ func (o *Operator) handleListChannels() {
 	t.Render()
 }
 
+func (o *Operator) handleListAppSessions() {
+	fmt.Println("Specify participant filter for app sessions (required):")
+	participant := o.readExtraArg("participant")
+	if !common.IsHexAddress(participant) {
+		fmt.Println("Invalid address format. Please provide a valid Ethereum address.")
+		return
+	}
+
+	fmt.Println("Specify status filter for app sessions (leave empty for no filter):")
+	statusSuggestions := []prompt.Suggest{
+		{Text: "open", Description: "List only open sessions"},
+		{Text: "closed", Description: "List only closed sessions"},
+	}
+	statusFilter := o.readSelectionArg("status", statusSuggestions)
+
+	offset := initialQueryOffset
+	for {
+		res, err := o.clearnode.GetAppSessions(participant, statusFilter, offset, defaultPageSize)
+		if err != nil {
+			fmt.Printf("Failed to get app sessions: %s\n", err.Error())
+			return
+		}
+
+		fmt.Printf("App Sessions for participant %s:\n", participant)
+		for _, session := range res.AppSessions {
+			t := table.NewWriter()
+			t.SetOutputMirror(os.Stdout)
+
+			t.AppendRow(table.Row{"ID", session.AppSessionID, session.AppSessionID, session.AppSessionID}, table.RowConfig{AutoMerge: true})
+			t.AppendRow(table.Row{"NAME", "", "PROTOCOL", session.Protocol}, table.RowConfig{AutoMerge: true})
+			t.AppendRow(table.Row{"NONCE", session.Nonce, "CHALLENGE", session.Challenge}, table.RowConfig{AutoMerge: true})
+			t.AppendRow(table.Row{"VERSION", session.Version, "STATUS", session.Status}, table.RowConfig{AutoMerge: true})
+			t.AppendRow(table.Row{"CREATED_AT", session.CreatedAt, "UPDATED_AT", session.UpdatedAt}, table.RowConfig{AutoMerge: true})
+			t.AppendSeparator()
+
+			if len(session.ParticipantWallets) > len(session.Weights) {
+				fmt.Printf("Warning: Mismatched participant wallets and weights in session %s\n", session.AppSessionID)
+				session.Weights = append(session.Weights, make([]int64, len(session.ParticipantWallets)-len(session.Weights))...)
+				continue
+			}
+
+			for i := range session.ParticipantWallets {
+				t.AppendRow(table.Row{fmt.Sprintf("PARTICIPANT_%d", i+1), session.ParticipantWallets[i], fmt.Sprintf("WEIGHT_%d", i+1), fmt.Sprintf("%d/%d", session.Weights[i], session.Quorum)}, table.RowConfig{AutoMerge: true})
+			}
+
+			t.Render()
+		}
+
+		fmt.Println("Do you want to see the next page?")
+		actionSuggestions := []prompt.Suggest{
+			{Text: "yes", Description: "Show next page"},
+			{Text: "no", Description: "Quit listing"},
+		}
```

Comment:
Maybe it is also worth adding "Previous page" option to give ability to go back and forth?